packages/layout/side-bar/SIDE-BAR.md

```diff
@@ -1,5 +1,7 @@
 # Side bar
-Collapsable navigation with icons and text.
+Collapsable navigation with icons and text. 
+
+The side bar is intended for applications. It must not be combined with the [mega menu](/layout/mega-menu) that is intended for websites. Use [app layout](/layout/app-layout) to give your application the right structure.
```

Comment:
It must not be combined with the [mega menu](/layout/mega-menu), which is intended for websites.